features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/org.wso2.carbon.identity.event.server.feature.default.json

```diff
@@ -170,7 +170,10 @@
   "identity_mgt.events.schemes.identityOauthEventHandler.module_index": "28",
   "identity_mgt.events.schemes.identityOauthEventHandler.subscriptions": [
     "POST_SET_USER_CLAIMS",
-    "POST_SET_USER_CLAIM"
+    "POST_SET_USER_CLAIM",
+    "POST_UPDATE_USER_LIST_OF_ROLE",
+    "PRE_DELETE_ROLE",
```

Comment:
POST_DELETE_ROLE?

Reply:
Have to go with the `PRE_DELETE_ROLE` as after role deletion, cannot retrieve the user Id list to revoke tokens and terminate sessions.